pkg/bindings/kube/types.go

```diff
@@ -42,6 +42,8 @@ type PlayOptions struct {
 	LogDriver *string
 	// LogOptions for the container. For example: journald
 	LogOptions *[]string
+	// Replace - replace existing pods and containers
+	Replace *bool
```

Comment:
That screams for tests.  It seems `kube play --replace` is not tested on the remote clients or the tests are incomplete.